client/src/lib/constants.ts

```diff
@@ -16,10 +16,14 @@ export const CEK_ACCOUNT_NONCE_SEED_STRING = 'solarium_cek_account';
 // must match get_channel_address_with_seed in state.rs
 export const CHANNEL_NONCE_SEED_STRING = 'solarium_channel';
 
+export const USER_DETAILS_ACCOUNT_NONCE_SEED_STRING = 'solarium_userdetails_account';
+
 // The cluster reference for the sol-did client - used to generate
 // DIDs that can be unambiguously resolved to any solana cluster
 // e.g. did:sol:devnet:abcde
 export const DEFAULT_CLUSTER = ClusterType.development(); // localhost - override with process.env.CLUSTER
 
 export const MESSAGE_SIZE_BYTES = 512;
 export const DEFAULT_MAX_MESSAGE_COUNT = 8;
+
+export const DEFAULT_USER_DETAILS_SIZE = 512;
```

Comment:
Maybe we can make this dynamic to the # of entries you want to support. e.g. n * MAX_ALIAS * DID_LENGTH.

Reply:
assuming the addressbook is an encrypted array of channels, or similar, can we reliably predict the size of the encrypted blob from the amount of channels?